AMP-92742 added docs for out-of-session events and deviceId in config… (#167)
amplitude/amplitude-dev-center: includes/sdk-out-of-session-events-next-gen.md

> You can also track events as out-of-session. Out-of-session events have a `sessionId` of `-1` and aren't considered part of the current session, meaning they don't extend the current session. This might be useful if you are logging events triggered by push notifications, for example. You can mark events as out-of-session by setting the `EventOptions` `sessionId` to `-1` when calling `track(event, options)` or `identify(identify, options)`.

Reviewer: ```suggestion
You can also track events as out-of-session. Out-of-session events have a `sessionId` of `-1` and aren't part of the current session, which means they don't extend the current session. 

Consider out-of-session events if you, for example, log events that a push notification triggers. Set the `sessionId` to `-1` in `EventOptions` to mark an event as out-of-session  when you call `track(event, options)` or `identify(identify, options)`.
```

@justin-fiedler Some additional context about why a Push Notification triggered event is a good use case for out-of-session events would be helpful. I assume it's because the user can interact with the push notification after the session that triggered the push notification has expired?

Author: Thanks @markzegarelli. See my updated suggestion.

> You can also track events as out-of-session. Out-of-session events have a `sessionId` of `-1` and aren't considered part of the current session, meaning they don't extend the current session. This might be useful if you are logging events triggered by push notifications, for example. You can mark events as out-of-session by setting the `EventOptions` `sessionId` to `-1` when calling `track(event, options)` or `identify(identify, options)`.

Author: ```suggestion
You can also track events as out-of-session. Out-of-session events have a `sessionId` of `-1` and aren't part of the current session, which means they don't extend the current session. They also do not start a new session, or change the sessionId for subsequent events. A potential usage case is push notifications which are usually external to the customers app usage.

Set the `sessionId` to `-1` in `EventOptions` to mark an event as out-of-session  when you call `track(event, options)` or `identify(identify, options)`.
```
